docs/geedocs/answer/4485225.html

```diff
@@ -13,14 +13,14 @@
   <p class="sidebar-toc"><a href="../answer/4580164.html">Welcome to Portable 5.0</a></p>
   <p class="sidebar-toc"><a href="../answer/4594620.html">Portable Release Notes 5.0</a></p>
   <p class="sidebar-toc"><a href="../answer/3230746.html">Portable User Guide</a></p>
+  <p class="sidebar-toc"><a href="../answer/4485225.html" class="current-file">Portable Globe and Server 5.2</a></p>
```

Comment:
I can't mark it in the right spot, but the page title (line 28 in the new document) still says 4.4.